src/extensions/replay/sessionrecording.ts

```diff
@@ -644,6 +651,10 @@ export class SessionRecording {
             this._urlBlocklist = response.sessionRecording.urlBlocklist
         }
 
+        if (response.sessionRecording?.eventTriggers) {
+            this._eventTriggers = response.sessionRecording.eventTriggers
+        }
```

Comment:
what was interesting in testing this was that i had an existing session, received the trigger in decide so didn't record

then when it was triggered i continued the same recording, _after_ that everything makes sense since a new session waits for the trigger

but might lead to unexpected behaviour during roll-out

so we could say 

* if you get a decide response that has different triggers to those you have stored
* immediately rotate the session id

but that feels like it could be unexpected too... maybe just us being aware of that roll out interaction is enough

no strong opinion here unless it triggers one in you :)